### ISSUE TITLE
Brewery - need Brewing background Service

### ISSUE THREAD

**Issue by springframeworkguru** [sfg-beer-works/sfg-beer-works#52]
Need scheduled task to brew beer. If Inventory qty on hand is less than beer.qtyOnhand, brew quantityToBrew - update inventory. Use Spring tasks scheduler

**Comment by dfcoffin** [comment]
Shouldn't the task order beer to be brewed if the beer_inventory.quantity_on_hand equal or fall below the beer.min_on_hand, at which point the beer.quantity_to_brew should be scheduled?
(since we don't have the brewing process currently available, updating the inventory automatically is a temporary bridge, but beer should go through stages (could be defined as a time period).

How frequent should the re-order task check the current inventory?

**Comment by springframeworkguru** [comment]
Yes - that's correct. We can run it every 30 seconds - it won't be that costly.
